packages/mobile/src/onboarding/registration/NameAndPicture.tsx

```diff
@@ -79,7 +81,8 @@ function NameAndPicture({ navigation }: Props) {
 
     dispatch(setPromptForno(true)) // Allow forno prompt after Welcome screen
     ValoraAnalytics.track(OnboardingEvents.name_and_picture_set, {
-      includesPhoto: false,
+      includesPhoto: photoChosen,
```

Comment:
this was just hardcoded to false before? weird..

Reply:
i know...